tests/index.js

```diff
@@ -17,6 +17,9 @@ const assert = require("chai").assert,
 // Tests
 //------------------------------------------------------------------------------
 
+const FIXABLE_MSG =
+":wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.";
```

Comment:
```suggestion
    ":wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.";
```

Let's add an indent for the continuation here. (Sadly, I need to update my ESLint config to enforce this)

Reply:
Fixed.